### PR TITLE
feat(pin-tabs): Added pin tabs functionality

### PR DESCRIPTION
**Pull request by pavlovicnemanja**

### CODE REVIEW

**Review by tomaskikutis — COMMENTED** [review]
Here are some issues I have discovered:

https://user-images.githubusercontent.com/2076953/106739543-9120e600-6619-11eb-9e0e-9ac5370a2ec0.mp4

https://user-images.githubusercontent.com/2076953/106739553-94b46d00-6619-11eb-8afd-9bff2e4d3abc.mp4

https://user-images.githubusercontent.com/2076953/106739561-967e3080-6619-11eb-9566-3f25d31918a5.mp4

File: scripts/core/ui/ui.ts [hunk 1]
```diff
@@ -915,19 +927,31 @@ function splitterWidget(superdesk, $timeout, $rootScope) {
...
                 start: function(e, ui) {
-                    workspace.resizable({maxWidth: container.width() - 730});
+                    workspace.resizable({maxWidth: container.width() - AUTHORING_MIN_WIDTH});
                 },
                 resize: resize,
                 stop: afterResize,
+                create: () => {
+                    // On double click handle, reset size to default
+                    angular.element('.ui-resizable-handle').dblclick(() => {
+                        workspace.removeAttr('style');
```
Comment (tomaskikutis) [inline scripts/core/ui/ui.ts:947]: Can a specific style be removed instead of all inline styles?

**Comment by tomaskikutis** [comment]
I have noticed one more bug - scrollbar for authoring view [disappears](https://user-images.githubusercontent.com/2076953/107206284-1aac2b80-69ff-11eb-8617-cde4950e7e84.png) after pinning a tab.

File: scripts/core/ui/ui.ts (hunk 1 above)
Comment (pavlovicnemanja) [reply scripts/core/ui/ui.ts]: done :)

**Comment by pavlovicnemanja** [comment]
@tomaskikutis except for this scrollbar, for which we can create the following ticket, can we merge this as discussed on scrum? tnx 🙂

**Comment by tomaskikutis** [comment]
It would be better to fix the scrollbar, but if we need it urgently and the fix would take a lot of time, then sure - merge this and open another PR for the fix.

**Review by fritzSF — APPROVED** [review]

**Pull request merged**